### PR TITLE
Fix i18n Pods relationship AJAX search

### LINKED ISSUE #5888: Relationship AJAX results doesn't use current post/term
We've created code to integrate with Polylang and WPML for the current post/term so the results are based on the current post/term language.
This works great, though the relationships AJAX results do not filter this the same way.

Due to this, the AJAX search results might be a different language depending on the current active language filter.

### PR DESCRIPTION
**Pull request by JoryHogeveen**
Fixes #5888 

## Description

Fixes the AJAX results for relationship fields based on query.

## PR Checklist

- [x] My code is tested.
- [x] My code follows the WordPress code style.
- [x] My code follows has proper inline documentation.
- [ ] My code includes automated tests for PHP and/or JS (if applicable).

### CODE REVIEW

File: classes/PodsI18n.php [hunk 1]
```diff
@@ -226,12 +226,12 @@ private static function default_strings() {
 	 */
 	public function get_current_language( $args = array() ) {
 
-		$args = wp_parse_args(
-			$args, array(
-				'refresh' => false,
-			)
+		$defaults = array(
```
Comment (tr1b0t) [inline classes/PodsI18n.php:229]: :no_entry_sign: Short array syntax must be used to define arrays

<sub>hash: 5659d632ad72cec2752380a50f6354aa</sub>

File: classes/PodsI18n.php [hunk 2]
```diff
@@ -256,12 +256,12 @@ public function get_current_language( $args = array() ) {
 	 */
 	public function get_current_language_data( $args = array() ) {
 
-		$args = wp_parse_args(
-			$args, array(
-				'refresh' => false,
-			)
+		$defaults = array(
```
Comment (tr1b0t) [inline classes/PodsI18n.php:259]: :no_entry_sign: Short array syntax must be used to define arrays

<sub>hash: 47b8fd00226044acb39760c425e9c29c</sub>

File: classes/PodsI18n.php [hunk 3]
```diff
@@ -313,15 +313,26 @@ public function get_current_language_data( $args = array() ) {
 				$current_language = get_user_meta( get_current_user_id(), 'pll_filter_content', true );
 			}
 
+			$pods_ajax = pods_v( 'pods_ajax', 'request', false );
+
 			// Get current language based on the object language if available.
 			$page = basename( $_SERVER['SCRIPT_NAME'] );
 
+			if ( $pods_ajax && 'admin-ajax.php' === $page ) {
+				$page = basename( $_SERVER['HTTP_REFERER'] );
```
Comment (tr1b0t) [inline classes/PodsI18n.php:322]: * :warning: Detected usage of a non-sanitized input variable: $_SERVER['HTTP_REFERER']
* :no_entry_sign: Detected usage of a possibly undefined superglobal array index: $_SERVER['HTTP_REFERER']. Use isset() or empty() to check the index exists before using it

<sub>hash: cd3e0b60c619c6310a3c7214ddf2147c</sub>

**Review by tr1b0t — COMMENTED** [review]
:no_entry_sign: 3 error(s)
:warning: 1 warning(s)

**Review by sc0ttkclark — APPROVED** [review]

**Pull request merged**